pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-JIRA: {{jira ticket number}}
+# JIRA: [jira ticket number]
```

Comment:
How about changing [Jira ticket number] to `[DTP-{jira ticket number}](https://bluedrop360.atlassian.net/browse/DTP-{jira ticket number})`?

I find it a bit annoying to go back to jira each time to copy the ticket link

Reply:
A bot will replace `[DTP-xxx]` by the correct link to the jira ticket automatically. It's not required.